android/app/build.gradle

```diff
@@ -22,7 +22,7 @@ if (flutterVersionName == null) {
 }
 
 apply plugin: 'com.android.application'
-apply plugin: 'com.google.gms.google-services'
+// TODO: turn firebase back on once needed apply plugin: 'com.google.gms.google-services'
 apply plugin: 'kotlin-android'
 apply from: "$flutterRoot/packages/flutter_tools/gradle/flutter.gradle"
 
```

Comment:
In my setup I cannot build without updated versions to:
1. compileSdkVersion 31
2. targetSdkVersion 31

Reply:
After reading https://medium.com/androiddevelopers/picking-your-compilesdkversion-minsdkversion-targetsdkversion-a098a0341ebd
It seems that generally the latest SdkVersion ought to be used for these parameters